scripts/parser.js

```diff
@@ -0,0 +1,49 @@
+const db = require('../app/database/dbConfig.js');
+var MBO = require('mindbody-sdk');
+const { insertJob, updateJobs } = require('../app/models/jobsModel.js');
+require('dotenv').config();
+ 
+var mbo = new MBO({
+    ApiKey: process.env.API_KEY, // from portal
+    SiteId: -99 //thisis the sandbox account
+});
+
+/**
+ * Iterates through provided class information from Mindbody API
+ * and updates or inserts jobs to the database
+ */
+async function processResponse(err,data){
+    if (err) {
+        console.error("Failed to retrieve information about classes", err);
+    } else {
+        classes = data.Classes
+        for (var i = 0; i < classes.length; i++){
```

Comment:
since classes is an array, in java, you can use the async function map with a call back function for any iterative objects
```await classes.map(async function(class_json) {

});
```